src/cr/cube/crunch_cube.py

```diff
@@ -737,6 +737,25 @@ def pairwise_pvals(self, axis=0):
         """
         return [slice_.pairwise_pvals(axis=axis) for slice_ in self.slices]
 
+    def compare_to_column(self, slice=0, column=0):
```

Comment:
I'm not sure we'll ever need this on the entire cube. It might be better to define this just on the slice level. Except maybe as an aggregation here, but definitely without the `slice` argument that you've provided here (unless I'm missing something obvious).